### DISCUSSION TITLE
MacOs-Setup

### DISCUSSION THREAD

**Discussion by giscus** [xy-241/CS-Notes#43] [General]
# MacOs-Setup

⚠️ Pre-requisites Package Manager Brew NPM Pipx Cargo Basher Daily Apps Floorp, a Firefox-based browser, Install Bypass Paywalls Arc - Chrome-based browser with great UX Google ...

https://notes.yxy.ninja/MacOs-Setup

**Comment by Edgar-Liu** [comment]
Hello XY,

Reminder to update the navi installation requirements! Cheers

**Reply by xy-241** [reply]
Hi Edgar, I have updated the navi installation guide! Let me know if there is anything else confusing :)
